WordPress/playground-tools
# Fix/wp now shared file mounting

## What?

This PR disables SQLite plugin updates and ensures that deleted files from the `wp-now` folder are added back on start.



## Why?



By clicking update on the SQLite plugin it would be deleted from the system wp-now folder. This would break wp-now on that machine.

## How?



By disabling updates of the SQLite plugin we will prevent users from breaking their environments.
For users that have broken environments, they will be automatically fixed on start by downloading missing folders.

## Testing Instructions


1. Checkout trunk
2. [Start a local wp-now env](https://github.com/WordPress/playground-tools/tree/trunk/packages/wp-now#contributing)
3. On the site go to the [Plugins screen](http://localhost:8881/wp-admin/plugins.php)
4. Click update on the `SQLite Database Integration` plugin
5. The update should fail and on reload your site will be broken
6. Check out the branch.
7. Rebuild and restart wp-now
8. Confirm that your site is working again
9.  On the site go to the [Plugins screen](http://localhost:8881/wp-admin/plugins.php)
10. Confirm that you can't update the `SQLite Database Integration` plugin
